html/canvas/offscreen/manual/the-offscreen-canvas/offscreencanvas.commit.html

```diff
@@ -14,20 +14,23 @@
     _assertPixel(canvas, 5,5, expectedR, expectedG, expectedB, expectedA, "5,5", expectedClrStr);
 }
 
-test(function() {
-    var placeholder = document.createElement('canvas');
-    placeholder.width = placeholder.height = 10;
-    var offscreenCanvas = placeholder.transferControlToOffscreen();
-    var ctx = offscreenCanvas.getContext('2d');
-    ctx.fillStyle = "#0f0";
-    ctx.fillRect(0, 0, 10, 10);
-    // commit() propagation is taken care of by an async task, which means the
-    // place holder contents should still be transparent black at this moment.
-    verifyPlaceholder(placeholder, 0,0,0,0, "0,0,0,0");
-    // Set timeout acts as a sync barrier to allow commit to propagate
-    setTimeout(function() {
-        verifyPlaceholder(placeholder, 0,255,0,255, "0,255,0,255");
-    }, 0);
+async_test(function(t) {
+    t.step(function() {
```

Comment:
You don't need this first `t.step`, the function passed to `async_test` is implicitly wrapped in a step and any exception thrown will fail the test.

Fixing this will also reduce the amount of indentation changes in this PR by a lot.